components/composer/src/index.html

```diff
@@ -58,15 +58,6 @@
   </head>
 
   <body>
-    <nylas-composer
-      id="demo-composer"
-      theme="dark"
-      template="Hey what up!<br/>
-      <br/>
-      <br/>
-      Thanks,
-      -Phil"
-    >
-    </nylas-composer>
+    <nylas-composer id="demo-composer" theme="dark"> </nylas-composer>
```

Comment:
Is there a reason why the template was removed? Could we add a default template back? Something that will make sense for UX.

Reply:
Yeah it's removed cause it doesn't work 😬 and it was breaking the Composer. I should make a ticket though,

Reply:
Was it breaking w/ a  TypeError `shadowRoot.getSelection()`? It isn't supported in firefox (and safari) and I've made a story for it https://app.shortcut.com/nylas/story/75902/html-editor-shadowroot-getselection-is-not-supported-in-firefox-and-safari